### PR TITLE
Rspect unit test

### PR DESCRIPTION
**Pull request by alicemirigo92**
This is a group project made by @Bigizi, @alicemirigo92, and @Habtamu210.
- Write unit tests using Rspec.
- Understand an existing codebase in a way that allows for adding tests.

### CODE REVIEW

**Comment by julie-ify** [comment]
Hi @alicemirigo92 and @Bigizi,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

### There are three collaborators for this project. Kindly ensure everyone contributes

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: Gemfile [hunk 1]
```diff
@@ -0,0 +1,5 @@
+ruby '3.2.2'
+
+gem 'rubocop', '>= 1.0', '< 2.0'
+source 'https://rubygems.org'
+gem 'rspec', '~> 3.12'
```
Comment (mukhammadrizooff) [inline Gemfile:5]: - [ ] [REQUIRED] Good job so far! As there are `4 failures` in your spec tests, please fix them 🙏🏼 Your tests should `not` have any failures 👍🏼 

image

**Review by mukhammadrizooff — CHANGES_REQUESTED** [review]
Hi @TEAM, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- Group work ✔️ 
- Correct GitFlow ✔️ 
- No linter errors ✔️ 
- Good implementations ✔️ 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by OyePriscilla — APPROVED** [review]
Hi @Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉
 


## Highlights
Write unit tests using `Rspec` ✔️ 
All test passed ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**